packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -73,3 +75,28 @@ export const initCoinjoinClient = (symbol: Account['symbol']) => async (dispatch
 // return only active instances
 export const getCoinjoinClient = (symbol: Account['symbol']) => () =>
     CoinjoinClientService.getInstance(symbol);
+
+// NOTE: this function will be extended in upcoming PR
+export const analyzeTransactions = (accountInfo: AccountInfo) => async () => {
```

Comment:
Does this have to be redux thunk? I mean, will there be any `dispatch` or `getState` call in upcoming PR?

Reply:
yes it will be a thunk action in the future